.gitignore

```diff
@@ -0,0 +1,30 @@
+
+# Default ignored files
+workspace.xml
+.gitkeep
+Acceptance.php
+acceptance.suite.yml
+AcceptanceTester.php
+AcceptanceTesterActions.php
+
+Functional.php
+functional.suite.yml
+FunctionalTester.php
+FunctionalTesterActions.php
+misc.xml
+modules.xml
+
+solutionAD/tests/_support/Helper/Unit.php
+unit.suite.yml
+UnitTester.php
+UnitTesterActions.php
+vcs.xml
+vēlreiz.iml
```

Comment:
File should contain relevant ignores for all, not some local user specific file exclude